pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/TableOp.java

```diff
@@ -44,6 +45,9 @@
   private String _schemaFileName;
   private Op _op;
   private String _tableConfigFileName;
+  private String _tableName;
+
+  private static final String CONTROLLER_URL = "localhost:9000";
```

Comment:
You may want to put this in another file, it will be needed in all the ops.

Reply:
Let me create a `CompatTestConstants` class, and put it in that file.